### ISSUE TITLE
Empty Dictionaries fail

### ISSUE THREAD

**Issue by adam-rocska** [mat-sz/plist#1]
If the file contains empty dictionaries (or empty stuff generally), like:
```
// !$*UTF8*$!
{
  archiveVersion = 1;
  classes = {};
```

The library fails with:
```
 throw new Error("Unexpected '".concat(char, "'"));
                ^

Error: Unexpected '}'
```

**Comment by mat-sz** [comment]
@adam-rocska Your example is parsing just fine, could you provide an example that reproduces the problem?

**Comment by mat-sz** [comment]
@adam-rocska Upon further analysis looks like this causes issues:

```
{
  classes = {
  };
}
```

Fixed in 1.1.0.
